packages/terra-table/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "terra-table",
+  "main": "lib/index.js",
+  "version": "4.36.0",
+  "description": "The terra-table component provides user a way to render selectable data in a tabular format",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/table-framework.git",
+    "directory": "packages/terra-table"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-table",
+    "Table",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/table-framework/issues"
+  },
+  "homepage": "https://github.com/cerner/table-framework#readme",
+  "peerDependencies": {
+    "react": "^16.8.5",
+    "react-dom": "^16.8.5",
+    "react-intl": "2 || 3 || 4 || 5"
+  },
+  "dependencies": {
+    "@cerner/terra-docs": "^1.9.0",
```

Comment:
I don't believe that terra-docs should be a dependency.  At most it is a devDependency, if needed.

Reply:
I don't think it should be needed at all.